suite-common/wallet-utils/src/stakingUtils.ts

```diff
@@ -56,16 +56,15 @@ export const getStakingLimitsByNetworkSymbol = (symbol: NetworkSymbol) => {
                 MIN_FOR_WITHDRAWALS: MIN_ETH_FOR_WITHDRAWALS,
                 MIN_BALANCE_FOR_STAKING: MIN_ETH_BALANCE_FOR_STAKING,
             };
-        case 'sol': {
+        case 'sol':
             return {
                 MIN_AMOUNT_FOR_STAKING: MIN_SOL_AMOUNT_FOR_STAKING,
                 MAX_AMOUNT_FOR_STAKING: MAX_SOL_AMOUNT_FOR_STAKING,
                 MIN_FOR_WITHDRAWALS: MIN_SOL_FOR_WITHDRAWALS,
                 MIN_BALANCE_FOR_STAKING: MIN_SOL_BALANCE_FOR_STAKING,
             };
-        }
         default:
-            throw new Error(`Unsupported network symbol: ${symbol}`);
+            return null;
```

Comment:
I think `throw` has its place here, as it can make future bugs easier to catch - at least keep a `console.warn`.

```suggestion
            console.warn(`Unsupported or missing network symbol: ${symbol}`);

            return null;
```